docs/hacking-guide/en/building.md

```diff
@@ -23,9 +23,9 @@ Install `NPM` using the instructions below
 
 The new npm-shrinkwrap.json should be written, commit it.
 
-#### Install npm On Fedora
+#### Install npm On Fedora 24
```

Comment:
Why not leave the old comment in, and add the new comment?  Not everyone's on Fedora 24 yet.